app/views/claims/_claimant.html.slim

```diff
@@ -35,14 +35,20 @@ fieldset
   = f.input :address_country, collection: ClaimantForm::COUNTRIES, include_blank: false
   = f.input :address_telephone_number, as: :tel, input_html: { class: 'medium-input' }
   = f.input :mobile_number, as: :tel, input_html: { class: 'medium-input' }
-  = f.input :email_address
 
-  fieldset
```

Comment:
You have removed the fieldset from the contact_preference question.  Not sure if this is intentional ? I think it should have the fieldset around it